<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,11">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_superior">
         <property name="styleSheet">
          <string notr="true">Qframe{
background-color:rgb(0, 0, 0);
}

QPushButton{
background-color:#000000ff;
border-radius:20px;
}

QPushButton:hover{
background-color:rgb(61, 61, 61);
border-radius:20px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="bt_menu">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../Downloads/lineas.jpeg</normaloff>../../Downloads/lineas.jpeg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 25 10pt &quot;Bahnschrift SemiLight&quot;;</string>
            </property>
            <property name="text">
             <string>INICIO</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bt_perfil">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../Downloads/Perfil.jpeg</normaloff>../../Downloads/Perfil.jpeg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_contenido">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(244, 242, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frame_menu">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>0</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
QFrame{
background-color: rgb(187, 134, 252);
border: 2px solid  rgb(126, 126, 126);
}

QPushButton{
background-color: white;
border-top-left-radius: 5px;
border-bottom-left-radius: 5px;
color: rgb(0, 0, 0);
font: 12pt &quot;Bahnschrift SemiLight&quot;;
}

QPushButton:hover{
background-color: rgb(61, 61, 61);
border-top-left-radius: 5px;
border-bottom-left-radius: 5px;
color: rgb(255, 255, 255);
font: 12pt &quot;Bahnschrift SemiLight&quot;;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="pushButton_2">
             <property name="geometry">
              <rect>
               <x>-10</x>
               <y>80</y>
               <width>211</width>
               <height>61</height>
              </rect>
             </property>
             <property name="text">
              <string>PushButton</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../../Downloads/Perfil.jpeg</normaloff>../../Downloads/Perfil.jpeg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
            </widget>
            <widget class="QFrame" name="frame_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>201</width>
               <height>81</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">
QFrame{
background-color: rgb(255, 255, 255);
}

QPushButton{
background-color: white;
border-top-left-radius: 5px;
border-bottom-left-radius: 5px;
color: rgb(0, 0, 0);
font: 12pt &quot;Bahnschrift SemiLight&quot;;
}

QPushButtom:hover{
background-color: rgb(61, 61, 61);
border-top-left-radius: 5px;
border-bottom-left-radius: 5px;
color: rgb(255, 255, 255);
font: 12pt &quot;Bahnschrift SemiLight&quot;;
}</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <widget class="QTextEdit" name="textEdit">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>10</y>
                <width>121</width>
                <height>61</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 9pt &quot;Bahnschrift SemiLight&quot;;
border-color: rgb(255, 255, 255);</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Bahnschrift SemiLight'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; color:#00007f;&quot;&gt;GRUPOS DE AMIGOS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
             <widget class="QPushButton" name="bt_mas">
              <property name="geometry">
               <rect>
                <x>140</x>
                <y>10</y>
                <width>51</width>
                <height>51</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../Downloads/suma.jpeg</normaloff>../../Downloads/suma.jpeg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_perfil">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item>
              <widget class="QStackedWidget" name="stackedWidget">
               <property name="styleSheet">
                <string notr="true">QFrame{
background-color: (255, 255, 255);
}

QLabel{
background-color: (255, 255, 255);
border: 1px solid  rgb(126, 126, 126);
color: rgb(85, 85, 255);
border-radius: 5px;
font: 10pt &quot;Bahnschrift SemiLight&quot;;
}</string>
               </property>
               <property name="currentIndex">
                <number>1</number>
               </property>
               <widget class="QWidget" name="page_perfil">
                <widget class="QPushButton" name="pushButton_3">
                 <property name="geometry">
                  <rect>
                   <x>572</x>
                   <y>557</y>
                   <width>161</width>
                   <height>40</height>
                  </rect>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-radius:20px;
border: 2px solid  rgb(143, 142, 142);
font: 25 10pt &quot;Bahnschrift SemiLight&quot;;
}

QPushButton:hover{
background-color:rgb(61, 61, 61);
border-radius:20px;
}</string>
                 </property>
                 <property name="text">
                  <string>ACTUALIZAR</string>
                 </property>
                </widget>
                <widget class="QSplitter" name="splitter">
                 <property name="geometry">
                  <rect>
                   <x>230</x>
                   <y>60</y>
                   <width>501</width>
                   <height>481</height>
                  </rect>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <widget class="QLabel" name="label">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_4">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_6">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Birthdate</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_5">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Gender</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_7">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Interests</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_8">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Work area</string>
                  </property>
                 </widget>
                </widget>
                <widget class="QSplitter" name="splitter_2">
                 <property name="geometry">
                  <rect>
                   <x>30</x>
                   <y>60</y>
                   <width>161</width>
                   <height>201</height>
                  </rect>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </widget>
               </widget>
               <widget class="QWidget" name="page_agregar">
                <widget class="QFrame" name="frame_3">
                 <property name="geometry">
                  <rect>
                   <x>29</x>
                   <y>109</y>
                   <width>701</width>
                   <height>491</height>
                  </rect>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
                <widget class="QSplitter" name="splitter_3">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>50</y>
                   <width>711</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QLineEdit" name="lineEdit">
                  <property name="styleSheet">
                   <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
border-radius:20px;
border: 2px solid  rgb(143, 142, 142);
font: 25 10pt &quot;Bahnschrift SemiLight&quot;;
}
</string>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="pushButton_4">
                  <property name="styleSheet">
                   <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-radius:20px;
border: 2px solid  rgb(143, 142, 142);
font: 25 10pt &quot;Bahnschrift SemiLight&quot;;
}

QPushButton:hover{
background-color:rgb(61, 61, 61);
border-radius:20px;
}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>../../Pictures/busqueda.jpg</normaloff>../../Pictures/busqueda.jpg</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>25</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                </widget>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
